### PR TITLE
Book detail I

### PR DESCRIPTION
**Pull request by mcolombo87**
## Summary

Texts and simple CSS for Book Details

## Screenshots

image

## Trello Card

https://trello.com/c/t5RQkePo/26-detalle-de-libro-i-textos

# Blocked By

#8

### CODE REVIEW

File: maquetado/scss/details.scss [hunk 1]
```diff
@@ -0,0 +1,25 @@
+.book-details {
+  p {
```
Comment (erikankodelia) [inline maquetado/scss/details.scss]: Don't use the tag to define props. Add class to the tags and use it here.

File: maquetado/scss/details.scss [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    display: block;
+    padding: 5px;
+  }
+  span {
```
Comment (erikankodelia) [inline maquetado/scss/details.scss]: Same here

File: maquetado/scss/details.scss [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    color: grey;
```
Comment (erikankodelia) [inline maquetado/scss/details.scss]: Create `colors` a file and add variables with the colors in hexa

File: maquetado/scss/details.scss [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  }
+}
+
+.book-title {
+  padding: 5px 5px 10px;
+}
+
+#title-genre {
+  color: grey;
```
Comment (erikankodelia) [inline maquetado/scss/details.scss]: Same here

File: maquetado/scss/details.scss [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+  font-weight: bold;
+}
+
+.title {
+  color: black;
+  font-weight: bold;
+  font-size: large;
```
Comment (erikankodelia) [inline maquetado/scss/details.scss]: Create a `sizes` file and add variables with the sizes in pixels

File: maquetado/scss/details.scss (hunk 1 above)
Comment (mcolombo87) [reply maquetado/scss/details.scss]: Done

File: maquetado/scss/details.scss (hunk 2 above)
Comment (mcolombo87) [reply maquetado/scss/details.scss]: Done

File: maquetado/scss/details.scss (hunk 3 above)
Comment (mcolombo87) [reply maquetado/scss/details.scss]: Done

File: maquetado/scss/details.scss (hunk 4 above)
Comment (mcolombo87) [reply maquetado/scss/details.scss]: Done!

File: maquetado/scss/details.scss (hunk 5 above)
Comment (mcolombo87) [reply maquetado/scss/details.scss]: Done

File: maquetado/scss/details.scss [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+    color: $grey;
+    font-size: $font-normal;
+  }
+}
+
+.book-title {
+  padding: 5px 5px 10px;
+}
+
+#title-genre {
```
Comment (erikankodelia) [inline maquetado/scss/details.scss]: Change to a class and not use the id.

File: maquetado/scss/details.scss [hunk 7]
```diff
@@ -0,0 +1,28 @@
+.book-details {
+  .book-item {
+    display: block;
+    padding: 5px;
+    font-size: $font-normal;
+  }
+  .book-item-span {
```
Comment (erikankodelia) [inline maquetado/scss/details.scss]: Add space between classes declaration

File: maquetado/scss/colors.scss [hunk 8]
```diff
@@ -0,0 +1,3 @@
+$grey: #828282;
+$black: #000000;
```
Comment (erikankodelia) [inline maquetado/scss/colors.scss]: You can change to $black: #000

**Pull request merged**